src/styleguide/_patterns/05-pages/03-locations-alt.json

```diff
@@ -0,0 +1,252 @@
+{
+  "title": "We are global",
+  "control": {
+    "label": "Menu"
+  },
+  "links": [
+    {
+      "url": "link.pages-about",
+      "control": {
+        "label": "About us"
+      }
+    },
+    {
+      "url": "https://medium.com/buildit",
+      "control": {
+        "label": "Blog"
+      }
+    },
+    {
+      "url": "link.pages-careers",
+      "control": {
+        "label": "Careers"
+      }
+    },
+    {
+      "url": false,
+      "control": {
+        "label": "Locations"
+      }
+    }
+  ],
+  "header": {
+    "url": "link.pages-homepage"
+  },
+  "footer": {
+    "copy": "made with ❤️"
+  },
+  "locations": [
+    {
+      "location": {
+        "name": "Bengaluru",
+        "addressline1": "72 Electronic City",
+        "addressline2": "Hosur Road",
+        "city": "",
+        "postcode": "Bangalore 560100",
+        "country": "India",
+        "img": "../../images/map/bengaluru.jpg",
+        "img-alt": "Bengaluru static map",
+        "map-link": "https://www.google.com/maps/place/Wipro+Limited/@12.838566,77.6592042,16z/data=!4m5!3m4!1s0x0:0x1d0caf77fe02554f!8m2!3d12.8384852!4d77.6571581?hl=en-US"
+      },
+      "control": {
+        "label": "Show on a map"
+      },
+      "jobs": [
```

Comment:
all `jobs` entries can be removed as they won't affect the page and will make the JSON a bit more readable! 😄